lib/engine/company.rb

```diff
@@ -59,6 +59,14 @@ def company?
       true
     end
 
+    def border?
```

Comment:
why are companies parts?

Reply:
There is no `Engine::Part::Blocker`, and this is the `Tile` method that gets called when adding a company as a blocker:

https://github.com/tobymao/18xx/blob/master/lib/engine/tile.rb#L250-L253

Reply:
In `View::Game::Part::Blocker`, `@blocker` is just a company everywhere